cdap-data-fabric/src/main/java/co/cask/cdap/spi/data/Row.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.spi.data;
+
+import javax.annotation.Nullable;
+
+/**
+ * Represents a row of data from a {@link StructuredTable}.
+ */
+public interface Row {
```

Comment:
In `cdap-api` module, there is also a `Row` interface. Can we have different name in here? Maybe `StructuredRow` since it will be coming from `StructuredTable`?